src/GitVersionCore.Tests/packages.config

```diff
@@ -8,8 +8,8 @@
   <package id="LibGit2Sharp.NativeBinaries" version="1.0.137" targetFramework="net45" />
   <package id="ModuleInit.Fody" version="1.5.9.0" targetFramework="net45" developmentDependency="true" />
   <package id="NSubstitute" version="1.10.0.0" targetFramework="net45" />
-  <package id="NUnit" version="3.2.1" targetFramework="net45" />
-  <package id="NUnitTestAdapter" version="2.0.0" targetFramework="net45" />
+  <package id="NUnit" version="3.4.1" targetFramework="net45" />
```

Comment:
Looks like you updated NUnit but didn't save the .csproj file before committing, which is why build is failing.